README.md

```diff
@@ -41,11 +41,11 @@ Web3 enhances the web we know today by making it decentralized, distributed, ope
 ---
 
 ### Summary
-Web1: Read
+Web1: Read :open_book:
```

Comment:
We, unfortunately, cant render emojis without Unicode on our website, can you please add Unicode instead?